lib/ui/components/reports/lists/ProjectSummaryList.svelte

```diff
@@ -1,64 +1,72 @@
 <script lang="ts">
-	import type { ProjectSummaryItem } from './types';
+  import type { ProjectSummaryItem } from "./types";
+  import { renderMarkdown } from "lib/util/renderMarkdown";
+  import { settingsStore } from "lib/util/stores";
 
-	export let data: ProjectSummaryItem[];
+  export let data: ProjectSummaryItem[];
 </script>
 
 <div class="time-entry-list">
-	<div
-		class="time-entry-group-header is-flex is-justify-content-space-between pb-1 mb-2"
-	>
-		<div class="is-flex is-align-items-center">
-			<span>Project</span>
-		</div>
-		<div class="is-flex">
-			<div class="header-right">Time</div>
-			<div class="header-right percent">Percent</div>
-		</div>
-	</div>
-	<div class="group-items">
-		{#each data as d}
-			<div class="group-item mb-2 is-flex is-justify-content-space-between">
-				<div class="is-flex is-align-items-center">
-					{#if d.hex}
-						<div class="project-circle mr-2" style="background-color:{d.hex}" />
-					{/if}
-					<span>
-						<span>{d.title ? d.title : '(No project)'}</span>
-						{#if d.client_title}
-							<span class="project-client">• {d.client_title}</span>
-						{/if}
-					</span>
-				</div>
+  <div
+    class="time-entry-group-header is-flex is-justify-content-space-between pb-1 mb-2"
+  >
+    <div class="is-flex is-align-items-center">
+      <span>Project</span>
+    </div>
+    <div class="is-flex">
+      <div class="header-right">Time</div>
+      <div class="header-right percent">Percent</div>
+    </div>
+  </div>
+  <div class="group-items">
+    {#each data as d}
+      <div class="group-item mb-2 is-flex is-justify-content-space-between">
+        <div class="is-flex is-align-items-center">
+          {#if d.hex}
+            <div class="project-circle mr-2" style="background-color:{d.hex}" />
+          {/if}
+          <span>
+            <span>
+              {#if $settingsStore.parseMarkdown}
+                {@html d.title ? renderMarkdown(d.title) : "(No project)"}
```

Comment:
I'm not sure if project titles should also be parsed as Markdown, only time entries make sense I think.

Reply:
I agree, I'll revert this bit and experiment with the suggestion below.